code/helper.go

```diff
@@ -34,6 +34,10 @@ func ExtractEnvironmentVariables() {
 	}
 
 	scrubRegex = os.Getenv("LM_SCRUB_REGEX")
+
+	logSource = "AWS"
```

Comment:
I think we should try to be more specific here as well. I could be on AWS but using different integration. Perhaps `lm-logs-aws`?

Reply:
Done